lstchain/reco/dl1_to_dl2.py

```diff
@@ -153,15 +151,15 @@ def train_disp_sign(train, custom_config={}, predict_feature='disp_sign'):
     """
 
     config = replace_config(standard_config, custom_config)
-    classification_args = config['random_forest_classifier_args']
-    features = config["classification_features"]
+    regression_args = config['random_forest_regressor_args']
```

Comment:
I think this was all correct before, right? Why rename the configuraition to regressor?

Reply:
no, the classification features in the config are meant for the gamma/hadron classification (include the reco energy and reco position)

Reply:
but, you are right for the args, that must be for a classifier!